charts/operator-wandb/charts/app/templates/deployment.yaml

```diff
@@ -134,6 +134,10 @@ spec:
                   name: {{ include "app.fullname" . }}-config
                   key: SLACK_SECRET
                   optional: true
+            {{- if ne .Values.global.email.host "" }}
```

Comment:
Since this is just related to the app, shouldn't the value be namespaced there instead of global?

Reply:
Since we don't know if or when this will become its own service, or which services might need email functionality, it's safest to place it in the global configuration for now.